README.md

```diff
@@ -1,101 +1,95 @@
 # RukPak
 
-RukPak runs in a Kubernetes cluster and defines an API for installing cloud native bundle content. 
+RukPak runs in a Kubernetes cluster and defines an API for installing cloud native bundle content.
 
-## Introduction 
+## Introduction
 
-RukPak is a pluggable solution for the packaging and distribution of cloud-native content and supports advanced strategies for installation, updates, and policy.
-The project provides a content ecosystem for installing a variety of artifacts, such as Git repositories, Helm charts, OLM bundles, and more onto a Kubernetes cluster. 
-These artifacts can then be managed, scaled, and upgraded in a safe way to enable powerful cluster extensions. 
+RukPak is a pluggable solution for the packaging and distribution of cloud-native content and supports advanced
+strategies for installation, updates, and policy. The project provides a content ecosystem for installing a variety of
+artifacts, such as Git repositories, Helm charts, OLM bundles, and more onto a Kubernetes cluster. These artifacts can
+then be managed, scaled, and upgraded in a safe way to enable powerful cluster extensions.
 
-At its core, RukPak is a small set of APIs, packaged as Kubernetes CustomResourceDefinitions, and controllers that watch for those APIs. These APIs express what 
-content is being installed on-cluster and how to create a running instance of the content. 
+At its core, RukPak is a small set of APIs, packaged as Kubernetes CustomResourceDefinitions, and controllers that watch
+for those APIs. These APIs express what content is being installed on-cluster and how to create a running instance of
+the content.
 
-## Components 
+## Components
 
-RukPak is composed of a few primary APIs: `Bundle`, `Instance`, and `ProvisionerClass`
+RukPak is composed of a few primary APIs: `Bundle`, `BundleInstance`, and `Provisioner`. These APIs work together to
```

Comment:
Link to the header sections for each of these APIs?

Reply:
I still think that `Provisioner` shouldn't be documented as an "API" but rather a concept?

Reply:
I still think that `Provisioner` shouldn't be documented as an "API" but rather a concept?

Reply:
Yes, I updated the language to not use the term API.